contrib/styles.scss

```diff
@@ -1 +1,47 @@
 @import "../core/neat";
+
+$color-neat-blue: #53aee0;
+$color-neat-orange: #FF7C56;
+$color-white: #fff;
+
+*, *:before, *:after {
```

Comment:
Begin pseudo elements with double colons: `::`<br>Each selector in a comma sequence should be on its own single line